snipeit.sh

```diff
@@ -136,58 +131,103 @@ install_packages () {
 create_virtualhost () {
   {
     echo "<VirtualHost *:80>"
-    echo "  <Directory $webdir/$name/public>"
+    echo "  <Directory $APP_PATH/public>"
     echo "      Allow From All"
     echo "      AllowOverride All"
     echo "      Options +Indexes"
     echo "  </Directory>"
     echo ""
-    echo "  DocumentRoot $webdir/$name/public"
+    echo "  DocumentRoot $APP_PATH/public"
     echo "  ServerName $fqdn"
     echo "</VirtualHost>"
   } >> "$apachefile"
 }
 
+create_user () {
+  echo "* Creating Snipe-IT user."
+
+  if [ "$distro" == "ubuntu" ] || [ "$distro" == "debian" ] ; then
+    adduser --quiet --disabled-password --gecos '""' "$APP_USER"
+  else
+    adduser "$APP_USER"
+  fi
+
+  usermod -a -G "$apache_group" "$APP_USER"
+}
+
+run_as () {
```

Comment:
I might rename this run_as_webuser() or similar.  it's confusing when reading through the code to know that "run_as php -r blah blah blah" does not mean to run the command a user named php.

Reply:
Great point, I’ll make the change later tonight.